### PR TITLE
Add test covering content tagger linking content

### PR DESCRIPTION
**Pull request by RyanMacG**
This test gives confidence that related content associated via content
tagger is properly displayed on frontend.

### CODE REVIEW

File: spec/content_tagger/linking_related_content_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    find("input.new-base-path").set("/" + related_content_slug)
+    click_button "Add related item"
+    expect(page).to have_text(related_content_slug)
+    click_button "Update tagging"
+    expect(page).to have_text("Tags have been updated!")
+  end
+
+  def then_the_related_content_is_linked_to_from_the_guide
+    reload_url_until_match(@guide_url, :has_text?, related_content_title)
```
Comment (kevindew) [inline spec/content_tagger/linking_related_content_spec.rb]: should we have status codes in here? I guess it starts as 404?

File: spec/content_tagger/adding_taxon_to_external_content_spec.rb [hunk 2]
```diff
@@ -28,11 +28,10 @@ def and_there_is_a_published_taxon
   end
 
   def when_i_tag_the_guide_with_the_taxon
-    visit(Plek.find("content-tagger") + "/taggings/lookup")
-    fill_in "content_lookup_form_base_path", with: "/" + guide_slug
-    click_button "Edit page"
+    visit_tag_external_content_page(slug: guide_slug)
     select2(taxon_title, css: "#s2id_tagging_tagging_update_form_taxons")
     click_button "Update tagging"
+    expect(page).to have_text("Tags have been updated!")
```
Comment (kevindew) [inline spec/content_tagger/adding_taxon_to_external_content_spec.rb]: Should this be in a separate commit?

File: spec/content_tagger/linking_related_content_spec.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+feature "Adding related content to Publisher content", new: true, content_tagger: true, frontend: true, publisher: true do
+  include ContentTaggerHelpers
+  include PublisherHelpers
+
+  let(:guide_title) { "Transaction with related content " + SecureRandom.uuid }
+  let(:guide_slug) { "transaction-with-related-content-" + SecureRandom.uuid }
+  let(:related_content_title) { "Related transaction " + SecureRandom.uuid }
+  let(:related_content_slug) { "related-transaction-" + SecureRandom.uuid }
+
+  scenario "Adding a taxon to a publisher guide" do
```
Comment (kevindew) [inline spec/content_tagger/linking_related_content_spec.rb]: should taxon here be related content? (not sure if they're synonyms or not)

Comment (adrianclay) [reply spec/content_tagger/linking_related_content_spec.rb]: Good spot! That is incorrect.

File: spec/content_tagger/linking_related_content_spec.rb (hunk 1 above)
Comment (adrianclay) [reply spec/content_tagger/linking_related_content_spec.rb]: Good catch.  I'd forgotten we need a double check.

File: spec/content_tagger/adding_taxon_to_external_content_spec.rb (hunk 2 above)
Comment (adrianclay) [reply spec/content_tagger/adding_taxon_to_external_content_spec.rb]: I'll separate it out.
